hibernate-core/src/test/java/org/hibernate/procedure/internal/ProcedureCallImplTest.java

```diff
@@ -0,0 +1,39 @@
+package org.hibernate.procedure.internal;
+
+import java.util.stream.Stream;
+import javax.persistence.EntityManager;
+import javax.persistence.Query;
+import org.hibernate.dialect.H2Dialect;
+import org.hibernate.jpa.test.BaseEntityManagerFunctionalTestCase;
+import org.hibernate.testing.RequiresDialect;
+import org.hibernate.testing.TestForIssue;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @author Nathan Xu
+ */
+public class ProcedureCallImplTest extends BaseEntityManagerFunctionalTestCase {
+
+	@Test
+	@TestForIssue( jiraKey = "HHH-13644" )
+	@RequiresDialect( H2Dialect.class )
+	public void testNoNullPointerExceptionThrown() {
+
+		EntityManager em = getOrCreateEntityManager();
+
+		em.getTransaction().begin();
+
+		em.createNativeQuery("CREATE ALIAS GET_RANDOM_VALUE FOR \"java.lang.Math.random\";").executeUpdate();
```

Comment:
The stored procedure native SQL is vendor specific so I specify H2 as the required dialect. However, it seems the bug and fixing is not vendor specific so any one should be enough.

Reply:
@Sanne I solved another critical bug. Enjoy!